variables.tf

```
@@ -51,3 +51,9 @@ variable "tags" {
   type        = map(string)
   default     = {}
 }
+
+variable "dns_ttl" {
+  description = "The TTL of DNS recursive resolvers to cache information about this record."
+  type        = string
```

this can be changed to 
```suggestion
  type        = number
  default     = 60
```